libnmstate/nm/ipv6.py

```diff
@@ -48,10 +48,8 @@ def get_info(active_connection):
 
 def create_setting(config):
```

Comment:
IMHO it would make sense to explain why the IP6_CONFIG methods are chosen like this to ensure that we understand the code when we look at it again in the future.